src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/Microsoft.DotNet.ApiCompat.Task.csproj

```diff
@@ -23,10 +23,16 @@
     <Compile Include="..\..\Tasks\Common\Message.cs" LinkBase="Common" />
     <Compile Include="..\..\Tasks\Common\MessageLevel.cs" LinkBase="Common" />
   </ItemGroup>
+  <!-- Include MSBuild logger -->
+  <ItemGroup>
+    <Compile Include="..\..\Microsoft.DotNet.ApiSymbolExtensions\Logging\MSBuildLog.cs" />
+  </ItemGroup>
 
   <ItemGroup>
     <!-- ExcludeAssets="Runtime" is being set to avoid adding package references and dependencies of package and project references into the package. -->
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildPackageVersion)" ExcludeAssets="Runtime" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" ExcludeAssets="Runtime" />
+    <ProjectReference Include="..\..\Microsoft.DotNet.ApiSymbolExtensions\Microsoft.DotNet.ApiSymbolExtensions.csproj" ExcludeAssets="Runtime" />
```

Comment:
we have reference to `Microsoft.DotNet.ApiSymbolExtensions.csproj` in `Microsoft.DotNet.ApiCompatibility.csproj`. I think we can omit this line here